mi-scheduler/base/argo-workflows/mi-workflow-template.yaml

```diff
@@ -9,10 +9,8 @@ metadata:
 
 spec:
   serviceAccountName: argo
-  entrypoint: mi-workflow
-
   templates:
-    - name: mi-workflow
+    - name: mi-anaylse-workflow
```

Comment:
don't we keep two templates? One for general MI to collect all knowledge and one specific for kebechet (subset)?

Reply:
yes that's right, gonna fix it

Reply:
okay @pacospace so in the end the base `mi` workflow was extended with `ENTITIES` parameter that specifies which entities would be analysed. I've removed the kebechet workflow because it probably should not be here, it is redundant, and we can genrally make the same call from `thoth.common` with just specified parameter :)